src/modules/auth/ui/views/2fa-view.tsx

```
@@ -0,0 +1,91 @@
+                            )}
+                            <input
+                                type="text"
+                                inputMode="numeric"
+                                maxLength={1}
+                                value={digit}
+                                onChange={(e) => handleChange(e.target.value, index)}
+                                onKeyDown={(e) => handleKeyDown(e, index)}
+                                ref={(el) => { inputsRef.current[index] = el }}
+                                className="w-14 h-14 text-xl font-medium text-center border border-input rounded-xl outline-none focus:ring-2 focus:ring-primary transition-all"
+                            />
+                        </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)